src/blocks/gallery-carousel/deprecated.js

```diff
@@ -14,6 +14,7 @@ import metadata from './block.json';
  * WordPress dependencies
  */
 import { getColorClassName, RichText } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
```

Comment:
You'll catch that running the lint:js, but single imports goes before multiple imports (LN16 wil switch with LN17)